docs/hub-tutorials/join-mainnet.md

```diff
@@ -333,7 +308,46 @@ Once state sync successfully completes, the node will begin to process blocks no
 
 ### Quicksync
 
-Quicksync.io offers several  daily snapshots of the Cosmos Hub with varying levels of pruning (`archive` 1.4TB, `default` 540GB, and `pruned` 265GB). For downloads and installation instructions, visit the [Cosmos Quicksync guide](https://quicksync.io/networks/cosmos.html).
+Quicksync.io offers several  daily snapshots of the Cosmos Hub with varying levels of pruning (`archive` 1.4TB, `default` 540GB, and `pruned` 265GB). For downloads and installation instructions, visit the [Gaia Quickstart Docs](https://hub.cosmos.network/main/getting-started/quickstart.html) and check out the `Quicksync` tab.
+
+For original Quicksync.io instructions visit [Quicksync.io's Cosmos guide](https://quicksync.io/networks/cosmos.html).
+::::::
+
+:::::: tab Blocksync
+
+### Blocksync
+
+> **Note**: Blocksync will take a significant amount of time and is not reccommended unless there's a specific need to sync a node from the beginning of cosmoshub-4
+
```

Comment:
we should mention this is only possible if using previous versions of gaia since you have to change the versions at upgrade heights

Reply:
yeah I can remove the comment for now and create an issue to give more in depth instructions on this.